### ISSUE TITLE
Ablation study about loss function

### ISSUE THREAD

**Issue by kkIIun** [addb-swstarlab/OANet#11]
Hello,

I have one question.
Is ensuring vector independence related to performance improvement in OANet?

So, has OAet ever experimented with using MSE loss only?

Thanks!

**Comment by Ychanho** [comment]
Hello,
I guess you're talking about the soft orthogonality regularization (SOR) term in the paper, which ensures vector independence.
SOR is the additional loss that enables the model to learn to have orthogonality between each vector, and it actually improves the performance of OANet.

In the code, the loss that the model uses to train is shown below.

loss = (1-self.lamb)*F.mse_loss(output, target) + self.lamb*dot_loss

Note that giving a value of 0 for lamb is equivalent to using only the MSE loss, as the impact of the SOR term is 0.
In fact, considering the SOR loss as well as the MSE loss resulted in a performance improvement over using only the MSE loss.  
However, a higher value of lamb does not always lead to better performance, so we need to find an appropriate value to consider both MSE loss and SOR loss.

Thanks!
